backend/tests/routers/gameRouter.test.ts

```diff
@@ -0,0 +1,33 @@
+import supertest from 'supertest';
+import app from '../../src/app';
+
+const request = supertest(app);
+
+const game = {
+	name: 'Test Game',
+	description: 'Test Description',
+	platform: 'Test Platform',
+	releaseDate: '2023-04-13',
```

Comment:
Needs to be a complete date time. Otherwise we get a zod error.
```suggestion
	releaseDate: '2023-04-13T00:00:00.000Z',
```